.changeset/clever-maps-wait.md

```diff
@@ -0,0 +1,6 @@
+---
+"@lynx-js/template-webpack-plugin": minor
+"@lynx-js/react-rsbuild-plugin": minor
+---
+
+**BREAKING CHANGE:** Remove the `enableParallelElement` and `pipelineSchedulerConfig` options. Since thread element resolution is still in experimental stage and may have stability risks, it will be disabled by default after this change.
```

Comment:
Let's make it two lines for better readability

```suggestion
**BREAKING CHANGE:** Remove the `enableParallelElement` and `pipelineSchedulerConfig` options.

Since the thread element resolution is still in experimental stage and may have stability risks, it will be disabled by default after this change.
```